pybase/exceptions.py

```diff
@@ -90,17 +90,19 @@ def _handle_exception(self, main_client, **kwargs):
                 if self.region_client is not None:
                     # We need to make sure that a different thread hasn't already
                     # reestablished to this region.
-                    loc = self.region_client.host + b":" + self.region_client.port
+                    loc = self.region_client.host + ":" + self.region_client.port
```

Comment:
in both py2 and py3, the `self.region_client` fields will be unicode strings.

in py2, these will happily be concatenated with the native (byte) string `":"` bc py2 allows this
in py3: native string `":"` will be in unicode, so will still happily concatenate here

without this change, py3 would continue having this error: `TypeError: can only concatenate str (not "bytes") to str`

same for below change